tests/Integration/Behaviour/Features/Context/Domain/OrderShippingFeatureContext.php

```diff
@@ -165,6 +166,19 @@ public function changeOrderShippingAddressTo(string $orderReference, string $ord
         $this->getCommandBus()->handle(new ChangeOrderDeliveryAddressCommand($orderId, $newDeliveryAddressId));
     }
 
+    /**
+     * @When I change order :orderReference invoice address to :orderInvoiceAddress
+     *
+     * @param string $orderReference
+     * @param string $orderInvoiceAddress
+     */
+    public function changeOrderInvoiceAddressTo(string $orderReference, string $orderInvoiceAddress)
+    {
+        $orderId = SharedStorage::getStorage()->get($orderReference);
```

Comment:
```suggestion
        $orderId = (int) SharedStorage::getStorage()->get($orderReference);
```

Reply:
Not required since the stored value is already an int This same code is used for ALL the order tests so it would imply modifying them all with no added value